target/linux/realtek/files-6.12/drivers/net/dsa/rtl83xx/rtl839x.c

```diff
@@ -293,6 +293,21 @@ static inline int rtl839x_l2_port_new_sa_fwd(int p)
 	return RTL839X_L2_PORT_NEW_SA_FWD(p);
 }
 
+static int rtldsa_839x_get_mirror_config(const struct rtl838x_switch_priv *priv,
+					 struct rtldsa_mirror_config *config,
+					 int group, int port)
+{
+	config->ctrl = RTL839X_MIR_CTRL + group * 4;
+	config->spm = RTL839X_MIR_SPM_CTRL + group * 4 * priv->port_width;
+	config->dpm = RTL839X_MIR_DPM_CTRL + group * 4 * priv->port_width;
```

Comment:
Swap the usage of port_width work with constants. That variable naming is totally confusing and is used only in stp and mirroring functions. As stp functions need to be converted to some similar logic, this variable can be removed some day.